spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Dropout.scala

```diff
@@ -180,7 +180,8 @@ class Dropout[T: ClassTag](
         this.gradInput.cmul(noise)
       }
     } else {
-      throw new IllegalArgumentException("backprop only defined while training")
+      this.gradInput = gradOutput
+      // throw new IllegalArgumentException("backprop only defined while training")
```

Comment:
why remove this ?

Reply:
For unit tests. May be changed to warning.

Reply:
ok, changed